src/fuel/app/classes/model/v1/comment.php

```diff
@@ -246,5 +246,34 @@ public static function get_all_post_comments($post_id) {
 			return $ex->getMessage();
 		}
 	}
+
+	/*
+	* method use to get all comments of a author_id
+	* @param input is author_id
```

Comment:
The better doccument (just note):
@param $author_id The system id of a target user.